internal/cli/atlas/metrics/databases/databases.go

```diff
@@ -12,23 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package metrics
+package databases
 
 import (
 	"github.com/mongodb/mongocli/internal/cli"
 	"github.com/spf13/cobra"
 )
 
-func DatabasesBuilder() *cobra.Command {
+func Builder() *cobra.Command {
 	const use = "databases"
 	cmd := &cobra.Command{
 		Use:     use,
 		Aliases: cli.GenerateAliases(use),
 		Short:   "List available databases or databases measurements for a given host.",
```

Comment:
What are "databases measurements"? Can we change to "database metrics, such as ...." ?

Reply:
Please add an example.